### ISSUE TITLE
orchestrate doesn't work in 2015.8.1-122-gacff40c

### ISSUE THREAD

**Issue by bogdanr** [saltstack/salt#29230]
I have this simple orchestrate formula:

```
mysql_setup:
  salt.state:
    - tgt: 'c-HowToWeb-dev-database'
    - highstate: True
```
and since the update to 2015.8.1-122-gacff40c it outputs this nasty error:

```
salt-run state.orchestrate test2
[ERROR   ] An exception occurred in this state: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/salt/state.py", line 1594, in call
    **cdata['kwargs'])
  File "/usr/local/lib/python2.7/dist-packages/salt/loader.py", line 1491, in wrapper
    return f(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/salt/states/saltmod.py", line 250, in state
    if not m_state:
UnboundLocalError: local variable 'm_state' referenced before assignment

CloudHero_master:
----------
          ID: mysql_setup
    Function: salt.state
      Result: False
     Comment: An exception occurred in this state: Traceback (most recent call last):
                File "/usr/local/lib/python2.7/dist-packages/salt/state.py", line 1594, in call
                  **cdata['kwargs'])
                File "/usr/local/lib/python2.7/dist-packages/salt/loader.py", line 1491, in wrapper
                  return f(*args, **kwargs)
                File "/usr/local/lib/python2.7/dist-packages/salt/states/saltmod.py", line 250, in state
                  if not m_state:
              UnboundLocalError: local variable 'm_state' referenced before assignment
     Started: 20:09:39.429789
    Duration: 2237.866 ms
     Changes:   

Summary for CloudHero_master
------------
Succeeded: 0
Failed:    1
------------
Total states run:     1
Total run time:   2.238 s
```

**Comment by tbaker57** [comment]
This looks like a duplicate of #29110

cheers
Tim

**Comment by jfindlay** [comment]
Thanks for the report, @bogdanr.  @tbaker57 is correct that this is a duplicate.  It is also the reason that 2015.8.2 was not released.  This has been fixed in 2015.8.3, which will come out soon.
